### PR TITLE
correct naming for abifm desert dust params

### PR DESCRIPTION
**Pull request by amylu00**
## To-do

- pull from main once CLIMAParameters PR is merged before squash and rebasing

## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

File: box/Project.toml [hunk 1]
```diff
@@ -8,3 +8,6 @@ OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 StatsPlots = "f3b207a7-027a-5e70-b257-86293d7955fd"
 Thermodynamics = "b60c26fb-14c3-4610-9d3e-2d17fe7ff00c"
+
+[compat]
+CLIMAParameters = "0.8.6"
```
Comment (trontrytel) [inline box/Project.toml:13]: I vote to keep it at "0.8" and not "0.8.6" in our toml files. The parameters are updated very often and we don't want to pin ourselves to some old version.

**Review by trontrytel — APPROVED** [review]
LGTM! 

I'm still not sure about pinning it to `0.8.6`. I know that technically it is the correct thing to do. But in practice parameters are updated very often and this will soon be outdated.

**Comment by amylu00** [comment]
> LGTM!
> 
> I'm still not sure about pinning it to `0.8.6`. I know that technically it is the correct thing to do. But in practice parameters are updated very often and this will soon be outdated.

Oh sorry I thought you meant just for the CloudMicrophysics toml file. Should I change them to 0.8?

**Comment by amylu00** [comment]
git commit --allow-empty -m "Trigger Build"

**Pull request merged**